barretenberg/cpp/src/barretenberg/dsl/acir_format/acir_to_constraint_buf.cpp

```diff
@@ -27,15 +27,55 @@ namespace acir_format {
 
 using namespace bb;
 
-/**
- * @brief Deserialize `buf` either based on the first byte interpreted as a
-          Noir serialization format byte, or falling back to `bincode` if
-          the format cannot be recognized. Currently only `msgpack` format
-          is expected, or the legacy `bincode` format.
- * @note Due to the lack of exception handling available to us in Wasm we can't
- *       try `bincode` format and if it fails try `msgpack`; instead we have to
- *       make a decision and commit to it.
- */
+/// ========= HELPERS ========= ///
+
+uint256_t from_be_bytes(std::vector<uint8_t> const& bytes)
```

Comment:
I see you didn't introduce this method but what does "be" stand for? Also, if the method expects bytes.size() = 32 then (1) the method should be named accordingly and (2) it should probably just be an array of size 32...

I know this is not your doing :)